cv32/sim/uvmt_cv32/Makefile

```diff
@@ -78,16 +78,17 @@ RISCVDV_PKG  := $(PROJ_ROOT_DIR)/cv32/env/uvme_cv32/riscv-dv
 # TB source files for the CV32E core
 TBSRC_HOME  := $(PROJ_ROOT_DIR)/$(COREV_PROJ)/tb
 export TBSRC_HOME  = $(PROJ_ROOT_DIR)/$(COREV_PROJ)/tb
-TBSRC_PKG   := $(TBSRC_HOME)/tb_riscv/include/perturbation_defines.sv
-TBSRC_TOP   := $(TBSRC_HOME)/uvmt_cv32/uvmt_cv32_tb.sv
-TBSRC       := $(TBSRC_HOME)/uvmt_cv32/uvmt_cv32_tb.sv \
-               $(TBSRC_HOME)/uvmt_cv32/uvmt_cv32_dut_wrap.sv \
-               $(TBSRC_HOME)/core/tb_top.sv \
-               $(TBSRC_HOME)/core/riscv_wrapper.sv \
-               $(TBSRC_HOME)/core/mm_ram.sv \
-               $(TBSRC_HOME)/core/dp_ram.sv \
-               $(TBSRC_HOME)/tb_riscv/riscv_random_stall.sv \
-               $(TBSRC_HOME)/tb_riscv/riscv_random_interrupt_generator.sv
+# Looks like these are redundant redundant
+#TBSRC_PKG   := $(TBSRC_HOME)/core/tb_riscv/include/perturbation_defines.sv
+#TBSRC_TOP   := $(TBSRC_HOME)/uvmt_cv32/uvmt_cv32_tb.sv
```

Comment:
TBSRC_TOP is used to process:

RTLSRC_VLOG_TB_TOP	:= $(basename $(notdir $(TBSRC_TOP)))
RTLSRC_VOPT_TB_TOP	:= $(addsuffix _vopt, $(RTLSRC_VLOG_TB_TOP))

These both variables are used in vsim.mk. 
TBSRC_TOP definition needs to be uncommented.

Reply:
OK, I will push that change back onto this branch.